python_bitbankcc/utils.py

```diff
@@ -23,6 +23,13 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+def try_json_parse(response, logger):
+    try:
+        return response.json()
+    except:
+        logger.debug('Invalid JSON: "' + response.text + '"')
```

Comment:
```suggestion
        logger.debug('Invalid JSON: ' + repr(response.content))
```

- it treats binary as well
- quoting may be confusing